test cases/windows/9 vs module defs generated/exe.def

```diff
@@ -0,0 +1,2 @@
+EXPORTS
+    thisfunc
```

Comment:
This should be exefunc according to failing tests

Reply:
ah yes, I renamed this in the C file but not here apparently :/